app/modals/stacking/stacking-modal.tsx

```diff
@@ -93,6 +95,12 @@ export const StackingModal: FC<StackingModalProps> = props => {
     balance: selectAddressBalance(state),
   }));
 
+  const accountBalance = new BigNumber(balance?.balance ?? 0);
+
+  const shouldModifyTxFee =
+    accountBalance.isEqualTo(amountToStack) ||
+    accountBalance.minus(amountToStack).isLessThan(CONTRACT_CALL_FEE);
```

Comment:
No need to have two checks here - if the first fails, the second will always fail. Maybe just use the second one.